google-beta/resource_sql_database_instance_test.go

```diff
@@ -594,6 +594,30 @@ func TestAccSqlDatabaseInstance_basic_with_user_labels(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_with_private_network(t *testing.T) {
```

Comment:
nit: could it be called `TestAccSqlDatabaseInstance_withPrivateNetwork`?

Reply:
Done.